sympy/functions/combinatorial/tests/test_comb_factorials.py

```diff
@@ -291,13 +291,14 @@ def test_binomial():
     assert binomial(10, 10) == 1
     assert binomial(n, z) == 1
     assert binomial(1, 2) == 0
+    assert binomial(-1, 2) == 1
```

Comment:
Why this test? This gives `1` in the current master as well.

Reply:
Yes, actually this test gave same result in both branches. I added it as sanity check with positive `binomial value` for negative `n`.